### PR TITLE
[#10516] Features page: reduce indentation

### LINKED ISSUE #10516: Features page: reduce indentation
Reduce indentation here?
image

Also, maybe move the 'raised edge' a little bit to the left, like this?
image

### PR DESCRIPTION
**Pull request by moziliar**
Fixes #10516 

**Outline of Solution**

Added a wrapper div for each of the components to offset from the raise page img, and change the overall bootstrap offset class to one with smaller indentation.

Outcome:
image

### CODE REVIEW

File: src/web/app/pages-static/features-page/features-page.component.scss [hunk 1]
```diff
@@ -2,3 +2,7 @@ img {
   margin: 15px 0;
   max-width: 80%;
 }
+
+.inner {
```
Comment (hcwong) [inline src/web/app/pages-static/features-page/features-page.component.scss:6]: Nitpicking: But could use a more descriptive classname here

**Review by hcwong — COMMENTED** [review]
Pulled changes and looks fine 👍 , though I think it will be good if you could include screenshots of the change with UI related PRs.

File: src/web/app/pages-static/features-page/features-page.component.scss (hunk 1 above)
Comment (moziliar) [reply src/web/app/pages-static/features-page/features-page.component.scss:6]: Make sense. Will make it more meaningful.

**Comment by moziliar** [comment]
@hcwong have done the changes to PR description and the classname. Please take a look :)

File: angular.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
               ],
               "index": {
                 "input": "src/web/index.prod.html",
-                "output": "index.html",
+                "output": "index.html"
```
Comment (hcwong) [inline angular.json:66]: Was this accidentally committed?

**Review by hcwong — APPROVED** [review]
lgtm :D

File: angular.json (hunk 2 above)
Comment (moziliar) [reply angular.json:66]: seems to be from some part of the setup... it should be this way I think?
Btw there's a CLI thing at the bottom. What would that be?

**Review by wkurniawan07 — APPROVED** [review]

**Pull request merged**